doc/configuration.rst

```diff
@@ -403,14 +400,20 @@ set to
 :class:`NumberOfCodeLinesSortKey(src_dir) <sphinx_gallery.sorting.NumberOfCodeLinesSortKey>`,
 which sorts the files based on the number of code lines::
 
-    from sphinx_gallery.sorting import NumberOfCodeLinesSortKey
     sphinx_gallery_conf = {
         ...
-        'within_subsection_order': NumberOfCodeLinesSortKey,
+        'within_subsection_order': "NumberOfCodeLinesSortKey",
     }
 
-In addition, multiple convenience classes are provided for use with
-``within_subsection_order``:
+Note that you must provide a *class* here, not an instance of the class. Because of how
```

Comment:
Stupid question, but if this is a string, does it matter if this is a class or instance of a class?

Reply:
Yes, it must be a class because we instantiate it internally. Hence the emphasis on it being a class not an instance here